codalab/rest/interpret.py

```diff
@@ -563,10 +599,6 @@ def resolve_items_into_infos(items):
     return new_items
 
 
-def expand_raw_items(raw_items):
```

Comment:
I removed this function because it doesn't really prove much help in understanding (the line is simple enough to know its expanding all raw_items), moreover we need to get the item_to_source mapping